src/styles/view-switcher.scss

```diff
@@ -0,0 +1,62 @@
+.view-switcher {
+  display: flex;
+  justify-content: center;
+
+  // These are the exact margins required to vertically align the top of the
+  // view switcher with the top of the Hide Highlights button to its left,
+  // and the top of the first annotation card below the view switcher with the
+  // top of the New Page Note button to its left.
```

Comment:
Yep. I'm not sure exactly what would be required to achieve that though - for example I'm not sure whether the existing elements are aligned to a grid (and therefore the grid size could be a variable that we could use multiples of). If it's not aligned to a grid and some of the gaps are seemingly arbitrary then we might end up with a lot of difficult to name variables. So needs investigation.